UnleashClient/__init__.py

```diff
@@ -279,8 +279,10 @@ def is_enabled(self,
         """
         context = context or {}
 
-        # Update context with static values
-        context.update(self.unleash_static_context)
+        base_context = self.unleash_static_context.copy()
+        # Update context with static values and allow context to override environment
+        base_context.update(context)
+        context = base_context
```

Comment:
Fixes an issue where the base context would overwrite the token's environment. This probably isn't a major issue in practice but it needs to be solved to get the client specs working fast